simpleLooping/test_normalizeString.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from normalizeString import normalizeString
+
+class TestNormalizeString(unittest.TestCase):
```

Comment:
Where is the test where we have space in the middle? Ex: "Hello             world!"